jsonschema-validation.xml

```diff
@@ -213,6 +213,24 @@
                 <eref target="https://json-schema.org/draft/2020-12/meta/validation"/>.
             </t>
 
+            <section title="Keyword Independence">
+                <t>
+                    Schema keywords typically operate independently, without
+                    affecting each other's outcomes.
+                </t>
+                <t>
+                    For schema author convenience, there are some exceptions among the
+                    keywords in this vocabulary:
+                    <list>
+                        <t>
+                            "requireAllExcept", whose behavior is defined in terms of "properties"
+                            "minContains", whose behavior is defined in terms of "contains"
+                            "maxContains", whose behavior is defined in terms of "contains"
```

Comment:
these should be 3 separate `<t>`s, I think? the list is rather incomplete as well. a complete list may be beyond the scope of this issue.

Reply:
Thanks. I fixed the `<t>`s. The list includes all the everything relevant to the validation vocabulary. A similar list appears in the other vocabularies that have keywords that break keyword independence.